include/mbedtls/ecdh.h

```diff
@@ -40,14 +41,12 @@ extern "C" {
 #endif
 
 /**
- * Defines the source of the imported EC key:
- * <ul><li>Our key.</li>
- * <li>The key of the peer.</li></ul>
+ * Defines the source of the imported EC key.
  */
 typedef enum
 {
-    MBEDTLS_ECDH_OURS,
-    MBEDTLS_ECDH_THEIRS,
+    MBEDTLS_ECDH_OURS, /**< Our key. */
```

Comment:
Minor style issue: Please align the two comments.

Reply:
Again, I find these comments superfluous.

Reply:
aligned.